components/Tabs/Tabs.js

```diff
@@ -3,18 +3,23 @@ class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
     // this.element;
-    
+    this.element = element;
     // Get the custom data attribute on the Link
     // this.data;
+    this.data = this.element.dataset.tab;
     
     // Using the custom data attribute get the associated Item element
     // this.itemElement;
-    
+      this.itemElement = document.querySelector(`.tabs-item[data-tab ="${this.data}"]`);      
     // Using the Item element, create a new instance of the TabItem class
     // this.tabItem;
+    this.tabItem = new TabItem(this.tabItem);
+
     
     // Add a click event listener on this instance, calling the select method on click
-
+    this.element.addEventListener('click', () => {
+      this.select();
+    })
   };
 
   select() {
```

Comment:
Looks like you're almost there! Just fill out the select() method and the TabItem class. In case the instructions are unclear, you will want to remove all of the active classes on the other tabs, apply the active class to the current tab, and call the select method on the tabItem.